backend/backend/settings.py

```diff
@@ -14,15 +14,16 @@
 import environ
 from django.core.management.utils import get_random_secret_key
 
-env = environ.Env(
-    DEBUG=(bool, False),
-)
+env = environ.Env()
```

Comment:
Why not leave the default as False? That way in production we don't need to set an extra environment variable.